faq/databases-for-redis.mdx

```diff
@@ -0,0 +1,75 @@
+
+</Message>
+
+## How many Private Networks can I attach?
+ 
+Currently, you can connect one Private Network to each Redis™ Instance in the Scaleway Console. For Standalone Instances you can attach multiple Private Networks via the CLI or the API only.
+
+## Can I use any IP Range for my Private Network?
+ 
+For now, only the [RFC1918](https://datatracker.ietf.org/doc/html/rfc1918) IP subnets are accepted as a Redis™ Instance virtual IPv4.
+
+## How is the Redis™ Database's failover IP managed?
+
+The Redis™ controller is aware of the cluster topology and routes requests automatically to a healthy node. In case of unavailability of one of the cluster nodes, no downtime is to be expected.
+
+## What should I use as virtual IP? How do I calculate the CIDR netmask?
+In a cluster topology, you have to [provide a private IPv4 address](https://www.scaleway.com/en/docs/faq/databases-for-postgresql-and-mysql/#what-should-i-use-as-virtual-ip-how-do-i-calculate-the-cidr-netmask) to each of your nodes. For instance, if you have a 3 nodes cluster, you have to declare 3 IPs which are on the same L3 network. The 3 IPs must be able to connect to one another.
+
+## What is the logic behind the cluster mode?
+
+A Redis™ cluster contains a set of primary and secondary nodes. The cluster nodes use hash partitioning to split the keyspace into 16,384 key slots. Each master of the cluster is responsible for a subset of those slots. Each secondary node replicates the data of one of the primary ones. For example, on a Databases for Redis™ cluster consisting of three compute Instances hosting each a primary node and a secondary node for one of the other nodes. If one of the primary nodes fails, the remaining nodes hold a vote and elect one of the remaining secondary nodes as the new primary node. When the failing node rejoins the cluster, it automatically becomes a secondary node and begins to replicate the data of another primary node. 
+You can scale your cluster horizontally up to six nodes. Below is an example of configuration for a three nodes cluster:
+* Instance A contains hash slots from 0 to 5,500
+* Instance B contains hash slots from 5,501 to 11,000
+* Instance C contains hash slots from 11,001 to 16,383
+
+Each of the three Instances acts as a primary node and replicates one of the others as a secondary node.
+
+## What's the difference between High Availability (HA) and Cluster mode?
+
+-- TO UPDATE -- 
+
+A Redis™ Cluster consists of primary and secondary nodes. The cluster uses hash partitioning to split the keyspace into key slots. Each secondary node replicates a specific primary node and can be reassigned to replicate another primary or be elected to a primary node as needed. This is much better for scaling as the operation is spread across multiple nodes instead of having a single entry point.
```

Comment:
```suggestion
A Redis™ cluster consists of primary and secondary nodes. The cluster uses hash partitioning to split the keyspace into key slots. Each secondary node replicates a specific primary node and can be reassigned to replicate another primary or be elected to a primary node as needed. This is much better for scaling as the operation is spread across multiple nodes instead of having a single entry point.
```